source/features/more-file-links.tsx

```diff
@@ -7,21 +7,30 @@ import features from '.';
 import GitHubURL from '../github-helpers/github-url';
 
 function handleMenuOpening({delegateTarget: dropdown}: delegate.Event): void {
-	dropdown.classList.add('rgh-raw-file-link'); // Mark this as processed
+	dropdown.classList.add('rgh-more-file-links'); // Mark this as processed
 
 	const viewFile = select('a[data-ga-click^="View file"]', dropdown)!;
-	const {href} = new GitHubURL(viewFile.href).assign({route: 'raw'});
+	const getDropdownLink = (name: string, route: string): JSX.Element => {
+		const {href} = new GitHubURL(viewFile.href).assign({route});
+
+		return (
+			<a data-skip-pjax href={href} className="pl-5 dropdown-item btn-link" role="menuitem">
```

Comment:
`data-skip-pjax` should only be applied to `raw` links. The History (`/commits`) page is PJAX-enabled by GitHub and I don't see any harm to PJAX Blame page as well.

Reply:
> I don't see any harm to PJAX Blame page as well

As long as they work correctly:

1. Visit the PR files page
2. Click one of these links
3. Open a new tab with the same exact URL
4. Compare the two
	- Any differences?

Related:

- https://github.com/refined-github/refined-github/issues/4638

Reply:
I don't see any differences, even after navigating some more, so I think it's okay to AJAX those.

Reply:
It should work properly. I really wish GitHub make more links PJAX-enabled, I guess it may cost too much resources to compute the partial markup which is consumed by PJAX.